test/setup-dependencies.sh

```diff
@@ -2,15 +2,19 @@
 TEST_DIR=$(pwd)
 cd ../..
 # get websocket demo repository
-git clone https://github.com/jfclere/httpd_websocket
+if [ ! -d httpd_websocket ]; then
+    git clone https://github.com/jfclere/httpd_websocket
+fi
 cd httpd_websocket
 git pull --rebase
 mvn install || exit 1
 cp target/websocket-hello-0.0.1.war $TEST_DIR/tomcat/
 cd ..
 
 # get mod_cluster (Java/Tomcat part)
-git clone https://github.com/modcluster/mod_cluster
+if [ ! -d mod_cluster ]; then
+    git clone https://github.com/modcluster/mod_cluster
+fi
```

Comment:
Should we have warning message if we are using the existing one?

Reply:
I don't think so. My intention was to prevent an error when the script is ran again (cloning to a non-empty directory). Using an already present directory/repository is correct imho, the `pull --rebase` will ensure  the content is up-to-date.

Reply:
We will probably want to change this altogether an just use maven instead, so whatever here is fine.